ERRATA.md

```diff
@@ -80,3 +80,27 @@ This document includes errata for the [Activity Streams](https://www.w3.org/TR/a
   - Unlike `latitude` and `longitude`, the domain of the `altitude` term is the `Object` type. The `altitude` term should be included in the list of properties of an `Object`. Because `altitude` is primarily documented as a property of a `Place`, publishers should not include `altitude` on objects that are not of type `Place`, and consumers should accept objects with this property that aren't of type `Place`.
 
   - The domain of the `attributedTo` property is both `Link` and `Object`. `attributedTo` should be included in the list of properties of a `Link`.
+
+  - Example 79 has  an `Image` object with `width` and `height` properties, which
+    are only allowed on `Link` objects. One alternative is to use a `Link` object
```

Comment:
```suggestion
    only indicate `Link` in their Domain.
```

I dont believe there is any normative text anywhere in the as2 TR that justifies a conclusion that these properties "are only allowed on Link objects".

I think a more appropriate resolution to the confusion here would be to add `Image` or `Object` to https://github.com/w3c/activitystreams/blob/main/vocabulary/index.html#L5531

Reply:
Hmmm, I just re-read some stuff and I'm kind of convinced here that maybe the domain of `width` and `height` _should_ be `Image | Link`?  If you'll indulge the wonkery, I read this sentence from section 4.2 explaining link, and comparing examples 11 and 12:

> Formally, the former example establishes an unqualified direct relationship with the image resource [i.e. a bare URL] while the latter creates a [qualified, indirect relationship](http://patterns.dataincubator.org/book/qualified-relation.html) that allows additional properties about the relationship to be specified.

If we read example 80 as a "qualified, indirect" reference to the image (resizing it from a bigger note1.png to 16x16, and resizing note2.png for the 32x32 version), then it makes a lot more sense why width and height would be set there (although the example would support this interpretation more obviously if they were both resizing `note-256-by-256.png` 😅 ).  This would also explain why `icon` is a distinct property, as opposed to just an `image`-- it's a qualified relationship and representational hint (i.e. show the 16x16 or the 32x32 depending on the window size/CSS/etc).

Just a theory... will try to make the next issue triage if it's worth discussing further.